src/interfaces/user.type.ts

```diff
@@ -1,8 +1,28 @@
 type userType = {
-  github_display_name: String,
-  github_id: string,
   id: string,
+  username?: string,
+  github_id: string,
+  github_display_name: String | null,
   incompleteUserDetails: boolean,
+  roles: {
+    archived: boolean,
+    admin?: boolean,
+    super_user?: boolean,
+  }
+  isMember: boolean,
+  status?: string,
+  picture?: {
+    publicId: string,
+    url: URL,
+  }
+  first_name?: string,
+  last_name?: string,
+  company?: string,
```

Comment:
Where are we getting company data from?

Reply:
it depends upon the whether when user registered they have entered or not, and you can see it's an optional property

Reply:
in singup we never enter our company data I guess so why keep it.

Reply:
![image](https://user-images.githubusercontent.com/33935339/190340493-5962d248-d0e7-44ec-bd2e-d249ad0c868e.png)
sample user data

Reply:
api.realdevsqaud.com/members